src/jaybenne/jaybenne.cpp

```diff
@@ -130,26 +130,27 @@ TaskCollection RadiationStep(Mesh *pmesh, const Real t_start, const Real dt) {
 //! parameters associated with Jaybenne, and enrolls the data variables associated with
 //! this physics package.
 std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin, Opacity &opacity,
-                                            Scattering &scattering, EOS &eos) {
+                                            Scattering &scattering, EOS &eos,
+                                            std::string block_name) {
   auto pkg = std::make_shared<StateDescriptor>("jaybenne");
```

Comment:
Is this a consistent change for this line?
```suggestion
  auto pkg = std::make_shared<StateDescriptor>(block_name);
```

Reply:
no, the package is still called "jaybenne"

Reply:
Ah - I see.